README.md.template

```diff
@@ -351,7 +351,24 @@ $ solo node start
 
 # output is similar to example-1 
 ```
+## For Developer Working on Hedera Repo
 
+To set customized `settings.txt` file, edit the file
+`./resources/templates/settings.txt` before `node setup` command.
+
+To deploy node with local build hedera code, first run `./gradlew assemble` at the root directory of hedera repo, then
+  run the following command:
+```
+$ solo node setup --local-build-path node0=<path-to-hedera-repo>,node1=<path-to-hedera-repo>,node2=<path-to-hedera-repo>
```

Comment:
We need to support just a single path for all nodes as well. We could offer `*` as the optional wildcard node ID:

```
solo node setup --local-build-path *=<path-to-hedera-repo>/hedera-node/data
```

OR without the wildcard,

```
solo node setup --local-build-path <path-to-hedera-repo>/hedera-node/data
```

I would recommend that we should support an absolute path to the app JAR:

```
solo node setup --local-build-path ../hedera-services/platform-sdk/app/data --app PlatformTestingTool.jar --app-config path-to-test-json-1,path-to-test-json-2
```

Reply:
Updated the logic of parsing `--local-build-path` to support a default build path.
Also created separate flags for `app` and `app-config`